### ISSUE TITLE
Doesn't seem to be working with Youtube API 3

### ISSUE THREAD

**Issue by chookity-pokk** [squiter/ivy-youtube#20]

**Comment by chookity-pokk** [comment]
Tried multiple times and I keep getting the error "symbol's value as variable is void: API KEY" where the API KEY is the one I got from following the steps in the description and replacing 'nil' with that API key.

**Comment by oios** [comment]
@ragreenburg i also had that problem and what i did was type literally "API KEY" then save it , then reopen and now place the good API key and it works!!
Now i can search youtube video tags in my minibuffer! but when i press enter to play the video andddddd: 
error in process sentinel: apply: Invalid function: "searching..." . this is what i get so tell me if that happends to you as well i hope @squiter could help us out or someone ! :)) cheers folks

**Comment by squiter** [comment]
Hey folks, sorry for the late reply!

I'll take a look at this as soon as I can... I'm not using this package anymore so I didn't notice this problem... but I'll take a look

**Comment by oios** [comment]
thanks @squiter really apreciatted ! nice job in doing this greeting from mexico :))

**Comment by Kabouik** [comment]
I just had to prepend the API key with `'` in `M-x customize-variable ivy-youtube-key` to save it, else I was getting this "void" issue as well.

**Comment by oios** [comment]
@Kabouik  so u are now able to do search and log into youtube? cause as i mention before i did that and everything fine but it stops in "searching..." and then it dies with process sentinel

**Comment by Kabouik** [comment]
Yes, with `ivy-youtube` version 20220818.2146 (installed in Guix using the [`guix-emacs` channel](https://github.com/babariviere/guix-emacs)):

https://user-images.githubusercontent.com/7107523/199856317-c017e740-6090-4896-b6f4-64c060d2e373.mov

Then all I did was `M-x customize-variable ivy-youtube-key` and paste my API v3 key after a `'` character (which was not in the README here), and `M-x customize-variable ivy-youtube-play-at` and set it to use `mpv`.

It works beautifully so far, even after restarting my emacs daemon. However I think what would be even more awesome would be to use an approach similar to that of [ytfzf](https://github.com/pystardust/ytfzf), which does not require a Google account at all (and yet allows subscriptions from a list of channels URLs).

**Comment by oios** [comment]
thanks man! glad people still using this and more so , this issue thread! :3

**Comment by Kabouik** [comment]
Just curious, did adding the `'` before your API key solve your issue? In [that post](https://github.com/squiter/ivy-youtube/issues/20#issuecomment-1058730378), that is not exactly what you did. If that works for you, then it would likely just require a little change in the README.

**Comment by MiragianCycle** [comment]
> I just had to prepend the API key with `'` in `M-x customize-variable ivy-youtube-key` to save it, else I was getting this "void" issue as well.

this worked for me. MacOS Sanoma M1, Emacs 30.
